lib/tjbot.js

```diff
@@ -344,8 +345,8 @@ TJBot.prototype._sendAudioWatsonSpeechtoText = function(callback) {
     // https://www.ibm.com/watson/developercloud/speech-to-text/api/v1/#recognize_audio_websockets
     this._micRecognizeStream = this._stt.createRecognizeStream({
         content_type: 'audio/l16; rate=44100; channels=2',
-        continuous: false,
```

Comment:
Two reasons: 
- continuous false returns ONLY the first transcript and ends the pipe. This is not expected behaviour as we expect tj.listen to trigger our callback as new messages arrive.
- continuous flag will be deprecated by April 10. inactivity timeout works better as it will help solve the 30 second inactivity issue that previously caused the pipe to automatically close needing reconnection.
 https://www.ibm.com/watson/developercloud/doc/speech-to-text/relnotes.shtml#March2017